### ISSUE TITLE
[bug] Clicking change name tag while name tag editing window is opened locks the game

### ISSUE THREAD

**Issue by abenkovskii** [KSP-KOS/KOS#718]
Mod list: kOS, RemoteTech, HyperEdit.
Steps to reproduce:
1. launch a vessel with a probe core and a kOS part.
2. open name tag editing window of one part.
3. open name tag editing window of another part.
4. press cancel.
5. press escape.
6. now you should see something wierd instead of pause menu.
log: https://www.dropbox.com/s/dtdqet6zvrc9hdu/kOS_bug_log.txt?dl=0

**Comment by abenkovskii** [comment]
It looks like there are two bugs:
1. one prevents you form closing the first name tag editing window after you open the second one.
2. another messes up KSP gui and some other functions while text editing window is opened.
How to reproduce the second bug:
1. open name tag window
2. unfocus it and press escape.
3. You will see what I'm talking about.

**Comment by abenkovskii** [comment]
Update:
Mods: kOS
1. screenshot (in VAB, almost all buttons are not working):
![screenshot0](https://cloud.githubusercontent.com/assets/11031560/7142877/4958fccc-e2e4-11e4-87ce-fded2c50f4d4.png)
2. log: https://www.dropbox.com/s/c0tmos3q3xqe0gk/output_log_mutliple_tag_windows.txt?dl=0

**Comment by abenkovskii** [comment]
I moved a discussion about broken pause menu into a separate issue: #805

**Comment by Dunbaratu** [comment]
The problem seems to be caused by the fact that I just inserted the default window ID of 0 when telling Unity to open the window.  Therefore Unity gets confused into treating two different name tag windows as if they were the same window for the sake of keyboard focus, mouse events, and so on.

When I changed it to use a unique integer per nametag window, the bug went away.

I was made suspicious by the fact that both windows were drawn in the same location onscreen.  They are supposed to be calculating their position based on which part you clicked, and that didn't make sense why they were getting mangled together.
